packages/wonder-blocks-cell/src/util/types.ts

```diff
@@ -14,29 +14,6 @@ export type HorizontalRuleVariant = "full-width" | "inset" | "none";
  */
 type Accessory = React.ReactNode;
 
-/**
- * A subset of CSS Properties to allow overriding some of the default styles set
- * on the accessory wrapper (loosely based on StyleType).
- */
-export type AccessoryStyle = {
-    /**
-     * A subset of valid Spacing values.
-     */
-    minWidth?: 16 | 24 | 32 | 48;
-    /**
-     * To vertically align the accessory.
-     */
-    alignSelf?: "flex-start" | "flex-end" | "center";
-    /**
-     * To horizontally align the accessory.
-     */
-    alignItems?: "flex-start" | "flex-end" | "center";
-    /**
-     * To set spacing between child elements.
-     */
-    gap?: number | string;
-};
```

Comment:
Thanks for switching it over to use `StyleType` instead! In general, do we want to prefer using `StyleType` for styles, rather than only accepting a subset of the CSS property types?